api/server.py

```diff
@@ -65,6 +92,13 @@ def send_next_metapaths_to_rate():
 # TODO: Maybe post each rated meta-path
 @app.route("/rate-meta-paths", methods=["POST"])
 def receive_rated_metapaths():
+    """
+    Endpoint where the rated meta-paths are posted.
+
+    Meta-paths are formated like this:
+    {'id': 3,
+    'rating': 0.75}
```

Comment:
same format as in `next-meta-paths`-endpoint

Reply:
Is it necessary to send the strings back and forth?

Reply:
It makes backend life easier

Reply:
I think this will have really bad performance. We should throw the `path` attribute out later on.